test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/junit/MosaicSimulationRule.java

```diff
@@ -196,9 +196,12 @@ public MosaicSimulation.SimulationResult executeSimulation(Path scenarioDirector
 
     public MosaicSimulation.SimulationResult executeSimulation(Path scenarioDirectory, String config) {
         try {
+            File f = scenarioDirectory.resolve(config).toFile();
+            if (!f.exists() || !f.isFile()) {
+                throw new InstantiationException("Scenario config file not found.");
```

Comment:
If we're really nitpicky, this isn't an `InstantiationException` but rather something related to IO. With the way you're handling exceptions now, we get the same error treatment for `ObjectInstatiation` fails and IO "errors".

Seing as the `SimulationResult` has an exception field it might make sense to throw a different exception here.